src/main/java/com/scottlogic/sort/SortAuthorSurname.java

```diff
@@ -1,14 +1,14 @@
 package com.scottlogic.sort;
 
 import com.scottlogic.SortOrder;
-import com.scottlogic.SortStrategy;
+import com.scottlogic.strategy.Sort;
```

Comment:
I don't like the explicit 'strategy' package.
You don't want to put all your strategies into one package, it won't scale - imagine how many different things you'd have in here.

Better to have a sort/sorter package for sort stuff